Implemented a case sensitive logic in Cleaner

roottool opened the pull request:
## Issue fixed
#4 

## Description
I implemented a case sensitive logic in `src/cleaner.js`.

I executed a unit test using [ava](https://github.com/avajs/ava).
I introduced [esm](https://github.com/standard-things/esm).
[This reason is that ava needs it when ava resolves `export default`](https://github.com/avajs/ava/blob/master/docs/recipes/es-modules.md).

roottool committed:
Introduced ava and esm

roottool committed:
Wrote an unit test for Cleaner

roottool committed:
Implemented a case sensitive logic in Cleaner

roottool committed:
Added newline

roottool committed:
Added a test pattern

404-html commented:
Amazing! Can we get bad words base64 encoded? 
EDIT: like it's done here:
https://github.com/404-html/internet-cleaner/blob/master/src/const.js#L3

roottool commented:
No, I assumed only normal strings.
I will think of something for base64 encored.

roottool committed:
Introduced atob for ava

roottool committed:
Added a test pattern for base64 bad word

404-html commented on test/cleaner.js:
```diff
@@ -0,0 +1,40 @@
+const niceWord = Consts.NICE_WORD;
+
+test('LowerCase', t => {
+    const badWord = 'fuck';
```
Re: discussion on the main topic. I should be more specific. 
My point is this repository not to have strings like this one. Just to prevent this tool from being hit by itself :)
Encoding this to base64 or simply using `f*ck` instead should do the job.

roottool commented:
I added a test pattern for a bad word of base64 encoded.
I checked the test pattern is passed.
The cleaner in `src/cleaner.js` can clean decoded bad words base64 encoded.

If you want to get bad words base64 encoded, I recommend to use [btoa method](https://developer.mozilla.org/ja/docs/Web/API/WindowBase64/btoa).

EDIT:
I took a mistake about the cleaner logic only changed from an one bad word to an one nice word.
I fixed to clean up all bad words in the sentence.

roottool committed:
Changed to clean all bad words for sentences

404-html commented:
Thanks for your changes @roottool, I've merged them 👌 
My point was not to have bad words in the code at all. [I've removed them](https://github.com/404-html/internet-cleaner/commit/493a3915cb3e2c0940bf63258d92c52926d8a9c6) after merge, I hope it's fine with you 😃
